openfermioncirq/experiments/hfvqe/molecular_example_odd_qubits.py

```diff
@@ -0,0 +1,40 @@
+import os
+from typing import Tuple
+
+import numpy as np
+import openfermion as of
+import scipy as sp
+
+from openfermioncirq.experiments.hfvqe.gradient_hf import rhf_minimization
+from openfermioncirq.experiments.hfvqe.objective import (
+    RestrictedHartreeFockObjective, generate_hamiltonian)
+
+
+def make_h3_2_5() -> Tuple[RestrictedHartreeFockObjective, of.MolecularData, np.
+                           ndarray, np.ndarray, np.ndarray]:
+    # load the molecule from moelcular data
+    import openfermioncirq.experiments.hfvqe as hfvqe
```

Comment:
small nit. move this import to the top of the file.  In case we call this functions again for testing, which it looks like we might since it generates data, it would be good to only import hfvqe once.